linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -115,6 +115,22 @@ void Process ()
 				ProcessQueue ();
 				ProcessVirtualMethods ();
 			}
+
+			// deal with [TypeForwardedTo] pseudo-attributes
+			foreach (AssemblyDefinition assembly in _context.GetAssemblies ()) {
+				if (!assembly.MainModule.HasExportedTypes)
+					continue;
+
+				foreach (var exported in assembly.MainModule.ExportedTypes) {
+					if (!exported.IsForwarder)
+						continue;
+					var type = exported.Resolve ();
+					if (!Annotations.IsMarked (type))
+						continue;
+					Annotations.Mark (exported);
+					Annotations.Mark (assembly.MainModule);
```

Comment:
The commit message is: `Only mark the destnation (the type being forwarded to) not the origin. Allows competely removing System.Core (if no types are needed from it)`.

@spouliot do you remember why this change was made?